src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,39 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class LoginPage extends BasePage {
+    @FindBy(css="[type='submit']")
+    WebElement submitButtonLocator;
+    @FindBy(css="[type='email']")
+    WebElement emailField;
+    @FindBy(css="[type='password']")
+    WebElement passwordField;
+    public LoginPage(WebDriver givenDriver){
+        super(givenDriver);
+            }
+     public LoginPage provideEmail(String email){
+        emailField.sendKeys(email);
+        return this;
+     }
+     public LoginPage providePassword(String password){
+         passwordField.sendKeys(password);
+       return this;
+    }
+      public LoginPage submitBtn(){
+        submitButtonLocator.click();
+        return this;
+     }
+   /*  public void login(){
+        provideEmail("karkisuman0330@gmail.com");
+        providePassword("te$t$tudent");
+        submitBtn();
+    }*/
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.